packages/cf-style-provider/package.json

```diff
@@ -31,4 +31,4 @@
   "peerDependencies": {
     "react": "^0.14.9 || ^15.0.0-0 || ^16.0.0-0"
   }
-}
+}
```

Comment:
Oh, this is a JSON file and not a javascript file. Having a newline at EOF doesn't apply here since we'll never concat JSON files.

I knew my editor wouldn't betray me!